### ISSUE TITLE
Resources are not properly being loaded. 

### ISSUE THREAD

**Issue by VincitCarnes** [python/pythondotorg#1426]
I have javascript enabled and have tried viewing this page in both chrome and firefox. I have also disabled uBlock origin and tried doing a hard refresh. I am still able to download the versions of Python I was looking for. However, I am just submitting this ticket in the event that you are not already aware that the issue exists. 

![image](https://user-images.githubusercontent.com/46457001/56235096-8f98bf00-603b-11e9-815b-42feb78c3401.png)

**Comment by berkerpeksag** [comment]
Thanks! I've just purged cache. It should work now.

**Comment by berkerpeksag** [comment]
@ewdurbin is there a way to solve this on Fastly or Heroku's end? I created a command to purge Fastly cache, but I doubt this will be enough to solve this issue.

See #1420 and #1423 for previous reports.

**Comment by berkerpeksag** [comment]
@ewdurbin here's my WIP branch: https://github.com/python/pythondotorg/compare/master...berkerpeksag:fix-fastly-cache

**Comment by ewdurbin** [comment]
@berkerpeksag if you open a PR for that WIP I can comment inline with suggested improvements.

**Comment by dogweather** [comment]
I'm getting this as well although it was closed as fixed a few hours ago in #1448.
Can I help out with a web transaction monitor that'll test the live site and signal when it's back online?

**Comment by mwichmann** [comment]
I ran into this yesterday, and it's indeed fixed for me now.  Does anyone think what the site code reports is a problem? That is, claiming javascript is disabled, when that isn't the problem - at the time I verified that javascript in other pages was otherwise working normally.

**Comment by dogweather** [comment]
I agree, @mwichmann that false error message looks like a separate bug we should file and take care of. If I understand correctly, the problem is a 404 CSS URL.

**Comment by dogweather** [comment]
FYI a more permanent solution is being worked on in PR #1427

**Comment by gpshead** [comment]
This is still happening (which I assume you know given the update an hour ago).  The Python 2.7 source download page links to https://www.python.org/downloads/release/python-2716/ which gets a 404 on its stylesheet https://www.python.org/static/stylesheets/style.3dbbbf7ee488 .  2715 works, as does 2713, most of the other 27\d+ urls don't.
